packages/block-library/src/embed/index.js

```diff
@@ -144,9 +144,9 @@ export function getEmbedEdit( title, icon ) {
 		 * @param {string} html The preview HTML that possibly contains an iframe with width and height set.
 		 */
 		maybeSetAspectRatioClassName( html ) {
-			const previewDom = document.createElement( 'div' );
-			previewDom.innerHTML = html;
-			const iframe = previewDom.querySelector( 'iframe' );
+			const previewDocument = document.implementation.createHTMLDocument();
```

Comment:
The `title` argument is not optional in IE11:

https://developer.mozilla.org/en-US/docs/Web/API/DOMImplementation/createHTMLDocument#Parameters

Can be solved by just passing `''` empty string.

Reply:
Fix pushed up. So glad we have you to pick up on these things, I would not have discovered that!